src/plugins/discover/public/application/apps/main/components/doc_table/actions/columns.ts

```diff
@@ -89,9 +87,7 @@ export function getStateColumnActions({
   }
 
   function onRemoveColumn(columnName: string) {
-    if (capabilities.discover.save) {
-      popularizeField(indexPattern, columnName, indexPatterns);
-    }
+    popularizeField(indexPattern, columnName, indexPatterns, capabilities);
```

Comment:
I wonder if a field should get more `popular` when it is removed from table :) I'd assume it somehow lost popularity 😄